docs/version-compatibility.md

```diff
@@ -1,28 +1,28 @@
-First of all, we recognize that the transitions from Pydantic V1 to V2 has been and will be painful for many users.
-We're sorry about this pain :pray:, it was an unfortunate but unavoidable consequence of both the rewrite to use Rust and of correcting some design mistakes of V1.
+First of all, we recognize that the transitions from Pydantic V1 to V2 has been and will be painful for some users.
+We're sorry about this pain :pray:, it was an unfortunate, but unavoidable, consequence of both the rewrite to use Rust and of correcting some design mistakes of V1.
```

Comment:
```suggestion
We're sorry about this pain :pray:, it was an unfortunate but necessary step to correct design mistakes of V1.
```

I'd just focus on the design mistakes of V1. I think that's where most of the breaking changes are coming from anyway.